src/test_linked_list.py

```diff
@@ -0,0 +1,112 @@
+"""This tests the linked_list module."""
+import pytest
+
+INSTANTIATION_TABLE = [
+    ("new_linked_list"),
+]
+
+
+PUSH_TABLE = [
+    (3, 3),
+    (4, 4)
+]
+
+
+POP_TABLE = [
+    (7, 9),
+    (33, 10)
+]
+
+
+SIZE_TABLE = [
+    (795, 1),
+    ([2, 33], 2),
+    ([3, 4, 5], 3),
+]
+
+
+SEARCH_TABLE = [
+    (3, 7, 3, 2, 3),
+    (7, 7, 5, 5, 7),
+    (3, 2, 1, 3, 3),
+    (9, 8, 8, 9, 9),
+    (8, 8, 8, 8, 8),
+]
+
+REMOVE_TABLE = [
+    (3, 7, 3, 2, 7),
+    (7, 8, 7, 5, 8),
+    (5, 8, 7, 5, 7),
+]
+
+
+@pytest.mark.parametrize("list_name", INSTANTIATION_TABLE)
+def test_linkedlist(list_name):
+    """Test instantiation LinkedList class."""
+    from linked_list import LinkedList
+    list_name = LinkedList()
+    assert list_name.size_of_list == 0
+
+
+@pytest.mark.parametrize("val, result", PUSH_TABLE)
+def test_push(val, result):
+    """Test push method to add value to front of linked list."""
+    from linked_list import LinkedList
+    list_name = LinkedList(val)
+    assert list_name.head.val == result
```

Comment:
Both tests coming from your `PUSH_TABLE` are the same test twice: "what happens if I only push on one value?". Try pushing on more than one.